### PR TITLE
[BLD-48235] update forms customisation file

### PR DESCRIPTION
**Pull request by ElizaSapir**

### CODE REVIEW

File: docs/quick_start.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Use this Quick Start guide to get you up and running with a working AI or live c
 
 `platform :ios, '10.0'`
 
+>Note: M1 Mac is supported from [v3.10.0](https://logmein-bold-mobile.github.io/bold360-mobile-docs-ios/docs/release-notes/#version-3100).
```
Comment (OmerRahmany) [inline docs/quick_start.md:26]: Maybe this is also a good place to reference docs/faq/Support-for-M1-mac.md

File: docs/release-notes.md [hunk 2]
```diff
@@ -188,6 +188,7 @@ Release date: May 12, 2021
 - [Changed Pod Specs fetching from Bintray to S3](https://logmein-bold-mobile.github.io/bold360-mobile-docs-ios/docs/faq/JCenter-deprecation/).
 - [Avatar variant for Dark Mode can be configured](https://logmein-bold-mobile.github.io/bold360-mobile-docs-ios/docs/chat-configuration/ui-customization/dark-mode/#avatar-dynamic-image-suuport).
 - Improved `ReadMore` content Loading.
+- M1 Mac Support.
```
Comment (OmerRahmany) [inline docs/release-notes.md:218]: What about Xcode13 Support?

**Review by OmerRahmany — APPROVED** [review]

File: docs/quick_start.md (hunk 1 above)
Comment (ElizaSapir) [reply docs/quick_start.md:26]: it's done on this PR

File: docs/release-notes.md (hunk 2 above)
Comment (ElizaSapir) [reply docs/release-notes.md:218]: No need to mention

File: docs/faq/Support-for-M1-mac.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+layout: default
+title: M1 Mac Support
+parent: FAQ
+nav_order: 4
+# permalink: /docs/faq/pod-update-version
+---
+
+# M1 Mac Support
+
+The Bold360 iOS SDK supports the new Apple Silicon architecture found in the M1 Mac machines from [v3.10.0](https://logmein-bold-mobile.github.io/bold360-mobile-docs-ios/docs/release-notes/#version-3100).
```
Comment (tehilaR2) [inline docs/faq/Support-for-M1-mac.md]: "... found on M1 mac machines"

File: docs/faq/Support-for-M1-mac.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+>You can now use the Xcode simulator on M1 devices.
+
+Please make sure your Podfile is set with the right iOS version.
+
+```swift
+platform :ios, "10.0" // must be 10 and above
+```
+For M1 compiling iOS sample click [here](https://github.com/bold360ai/bold360-mobile-samples-ios/tree/master/m1Sample).
```
Comment (tehilaR2) [inline docs/faq/Support-for-M1-mac.md]: > M1 compatible [sample](https://github.com/bold360ai/bold360-mobile-samples-ios/tree/master/m1Sample)

File: docs/faq/Support-for-xcode13.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+# Xcode13 Support
+
+The Bold360 iOS SDK is supported under Xcode13 from [v3.10.0](https://logmein-bold-mobile.github.io/bold360-mobile-docs-ios/docs/release-notes/#version-3100).
+
+Please make sure your Podfile is set with the right iOS version.
+
+```swift
+platform :ios, "10.0" // must be 10 and above
+```
+For Xcode13 compiling iOS sample click [here](https://github.com/bold360ai/bold360-mobile-samples-ios/tree/master/m1Sample).
```
Comment (tehilaR2) [inline docs/faq/Support-for-xcode13.md]: same as before

**Review by tehilaR2 — APPROVED** [review]

File: docs/release-notes.md [hunk 6]
```diff
@@ -14,6 +14,31 @@ toc_float: true
...
+
+# Version 3.14.0
+Release date: October 14, 2021
+{: .overview}
+
+- Configurable custom fonts and text color on live forms.
+  
+### Fixes
+
+- Supporting the passing of AI channeling events to the reports.
```
Comment (OmerRahmany) [inline docs/release-notes.md]: Change to "Ensuring AI channeling events are passed on to reports"

File: docs/release-notes.md [hunk 7]
```diff
@@ -14,6 +14,31 @@ toc_float: true
...
+- Supporting channel ticket.
```
Comment (OmerRahmany) [inline docs/release-notes.md]: What does it mean?

File: docs/release-notes.md [hunk 8]
```diff
@@ -12,7 +12,34 @@ toc_float: true
...
+
+<summary> Version 3.14.0 </summary>
+
+# Version 3.14.0
+Release date: October 14, 2021
+{: .overview}
+
+### Features
+
+- Configurable custom fonts and text color on live forms.
```
Comment (OmerRahmany) [inline docs/release-notes.md]: Change to: "Text color and font can be configured for live forms."

File: docs/release-notes.md [hunk 9]
```diff
@@ -12,7 +12,34 @@ toc_float: true
...
+- Ticket typed channel with Linked article url.
+
+### Fixes
+
+- Bot channels reports weren't delivered.
```
Comment (OmerRahmany) [inline docs/release-notes.md]: Change to:
"Reports for bot channels were not delivered"

File: docs/release-notes.md [hunk 10]
```diff
@@ -12,7 +12,34 @@ toc_float: true
...
+<summary> Version 3.14.0 </summary>
+
+# Version 3.14.0
+Release date: October 14, 2021
+{: .overview}
+
+### Features
+
+- Configurable custom fonts and text color on live forms.
+- Ticket typed channel with Linked article url.
```
Comment (OmerRahmany) [inline docs/release-notes.md:26]: What does that mean?

**Pull request merged**